operations/mimir-rules-action/entrypoint.sh

```diff
@@ -73,8 +73,10 @@ case "${ACTION}" in
 esac
 
 SINGLE_LINE_OUTPUT=$(echo "${OUTPUT}" | awk 'BEGIN { RS="%0A" } { gsub(/%/, "%25"); gsub(/\r/, "%0D"); gsub(/\n/, "%0A") } { print }')
-echo ::set-output name=detailed::"${SINGLE_LINE_OUTPUT}"
+echo "detailed=${SINGLE_LINE_OUTPUT}" >> $GITHUB_OUTPUT
 SUMMARY=$(echo "${OUTPUT}" | grep Summary)
-echo ::set-output name=summary::"${SUMMARY}"
+echo 'summary<<ENDOFSUMMARY' >> $GITHUB_OUTPUT
```

Comment:
Note to reviewers: I thought we needed the `=` sign after `summary` here, but apparently it's not required: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#multiline-strings